examples/DateRangePickerWrapper.jsx

```diff
@@ -1,7 +1,69 @@
 import React from 'react';
+import moment from 'moment';
+import omit from 'lodash.omit';
 
 import DateRangePicker from '../src/components/DateRangePicker';
 
+import DateRangePickerShape from '../src/shapes/DateRangePickerShape';
+import { START_DATE, END_DATE, HORIZONTAL_ORIENTATION, ANCHOR_LEFT } from '../constants';
+import isInclusivelyAfterDay from '../src/utils/isInclusivelyAfterDay';
+
+const propTypes = omit(DateRangePickerShape, [
```

Comment:
It's a dev dependency so I didn't think it was a big deal. Doesn't using object rest require unused vars to be ignored?